.github/ISSUE_TEMPLATE/release_retrospective_template.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Release retrospective
+title: "[Retrospective] Release Version {{ env.VERSION }}"
+labels: untriaged, release, v{{ env.VERSION }}
+---
+
+### **Related release issue?**
+{{ env.RELEASE_ISSUE_URL }}
+
+### **How to use this issue?**
+Please add comments to this [Retro Board](https://github.com/orgs/opensearch-project/projects/205/views/4), they can be small or large in scope. Honest feedback is important to improve our processes, suggestions are also welcomed but not required.
```

Comment:
You should remove `views/4` as that defaults to `2.15.0` view.
https://github.com/orgs/opensearch-project/projects/205

On another hand, we probably can think about how to automate a view creation as well. Thanks.

Reply:
Project board has very restrictive permissions. How about just comment on the issue with the feedbacks and stuff? The release manager can take care of moving it to the board later down the line? 
cc: @getsaurabh02

Reply:
That is also probably a good approach.

Reply:
Updated the template. Thanks!